### DISCUSSION TITLE
Suggest using reduced frame rate for Pi Zero

### DISCUSSION THREAD

**Discussion by zeroepoch** [tiny-pilot/tinypilot#586] [Show and tell]
Reducing the framerate to 15 FPS greatly reduces the CPU usage (~50% total) on the Pi Zero W and improves the keyboard response time and helps with drop/repeated keys during my experiments. I'd recommend suggesting this on the wiki somewhere. I'm using the following settings with improved success over the default settings.

```
ustreamer_desired_fps: 15
ustreamer_drop_same_frames: 15
```

**Comment by mtlynch** [comment]
Thanks for sharing this!

Pi Zero is not officially supported, but a page on the wiki would be much appreciated!

**Reply by zeroepoch** [reply]
I created the following page: https://github.com/mtlynch/tinypilot/wiki/Pi-Zero-Optimizations. Please edit the page or do whatever you wish with it.

**Reply by mtlynch** [reply]
Awesome, thanks!

**Reply by zeroepoch** [reply]
Thanks @mtlynch for the improvements. I think we can close this now if that's what you do for discussions. Not sure what the procedure is compared to issues.
